html/pages/index.html

```diff
@@ -3,180 +3,259 @@
+
+    <h1 class="logo">
+      <a class="navbar-brand" href="#">Movies&amp;Stuff</a>
+    </h1>
+
+    <div class="collapse navbar-toggleable-sm" id="navbarResponsive">
+      <ul class="nav navbar-nav float-md-right">
+        <li class="nav-item">
+          <a class="nav-link" href="#movies">Movies</a>
+        </li>
+
+        <li class="nav-item">
+          <a class="nav-link" href="#tv-shows">TV Shows</a>
+        </li>
+
+        <li class="nav-item">
+          <a class="nav-link btn navbar-btn" href="#subscribe">Subscribe</a>
+        </li>
+      </ul>
+    </div>
+  </div>
 </nav>
 
+<!-- completely removed the entire row wrapping both the figure and the
+details-->
 <header class="featured">
-  <figure class="featured-figure">
-    <img src="images/deadpool-preview.jpg" alt="deadpool-preview">
-  </figure>
-
-  <div class="featured-movie">
-    <h2 class="featured-movie-title">Deadpool</h2>
-
-    <div class="featured-movie-rating">
-      <img src="images/rating-star.png" alt="a rating star-1">
-      <img src="images/rating-star.png" alt="a rating star-2">
-      <img src="images/rating-star.png" alt="a rating star-3">
-      <img src="images/rating-star.png" alt="a rating star-4">
-      <img src="images/rating-star.png" alt="a rating star-5">
-    </div>
+  <div class="container">
+    <div class="row">
+      <figure class="featured-movie-poster col-md-3 offset-md-2">
+        <img src="{{root}}images/deadpool-preview.jpg"
+             alt="deadpool-preview"
+             class="img-fluid float-xs-left">
+      </figure>
+
+      <section class="featured-movie col-md-7">
+        <h2 class="featured-movie-title">Deadpool</h2>
 
-    <p class="featured-movie-description"> A former Special Forces operative
-     turned mercenary is subjected to a rogue experiment that leaves him with
-     accelerated healing powers, adopting the alter ego Deadpool.
-    </p>
+        <div class="featured-movie-rating">
+          <img src="{{root}}images/rating-star.png" alt="a rating star-1">
+          <img src="{{root}}images/rating-star.png" alt="a rating star-2">
+          <img src="{{root}}images/rating-star.png" alt="a rating star-3">
+          <img src="{{root}}images/rating-star.png" alt="a rating star-4">
+          <img src="{{root}}images/rating-star.png" alt="a rating star-5">
+        </div>
 
-    <footer class="featured-movie-details">
-      R | 108 min | Action Adeventure, Comedy |
-      <time datetime="2016-02-12">12 February 2016</time>
-    </footer>
+        <p class="featured-movie-description">
+          A former Special Forces operative turned mercenary is subjected
+           to a rogue experiment that leaves him with accelerated healing
+           powers, adopting the alter ego Deadpool.
+        </p>
+
+        <footer class="featured-movie-details">
+          R | 108 min | Action Adeventure, Comedy |
+          <time datetime="2016-02-12">12 February 2016</time>
+        </footer>
+      </section>
```

Comment:
Our `figure` and `section` are closely related elements. So let's wrap them in one `article` tag.